src/System.Linq/src/System/Linq/ElementAt.cs

```diff
@@ -89,7 +91,7 @@ public static TSource ElementAtOrDefault<TSource>(this IEnumerable<TSource> sour
                 }
             }
 
-            return default(TSource);
+            return default!;
```

Comment:
We will not be adding TODO's in this effort to clean-up later? or we will just wait for an analyzer to do it all for us?

Reply:
I suggest we don't wait for an analyzer because according to the language team, this is really hard to do in the general case. We should use TODO markers.

Reply:
I've not added TODOs in places where I'm not expecting any follow-up. To my knowledge there's currently no plan to address this in the language. If/when it's supported, if there's no analyzer yet, we can search for `default!` .

Reply:
Yeah, that makes sense.